tests/base/time_stepping_01.cc

```diff
@@ -266,6 +266,32 @@ int
 main()
 {
   initlog();
+  deallog.precision(4);
```

Comment:
do you need this? This changes the output below.

Reply:
Yes, for the fifth-order scheme, two places of the output in release mode are different from debug mode, I think they are beyond the machine precision.
debug mode:
DEAL::l2 norm error: 2.95205e-08   4.93262
DEAL::l2 norm error: 9.43231e-10   4.96796
DEAL::l2 norm error: 2.97971e-11  **_4.98437_**
DEAL::l2 norm error: 9.33476e-13  **_4.99641_**
DEAL::l2 norm error: 2.79776e-14   5.06027

release mode:
DEAL::l2 norm error: 2.95205e-08   4.93262
DEAL::l2 norm error: 9.43231e-10   4.96796
DEAL::l2 norm error: 2.97979e-11   **_4.98433_**
DEAL::l2 norm error: 9.33476e-13   **_4.99646_**
DEAL::l2 norm error: 2.79776e-14   5.06027

Reply:
Keep it as is, ``numdiff`` hopefully takes care of the difference. Do you have ``numdiff`` installed?

Reply:
If I keep it, the test would fail.
1822: ----------------
1822: ##123     #:6   <== 4.98437
1822: ##123     #:6   ==> 4.98433
1822: @ Absolute error = 4.0000000000e-5, Relative error = 8.0251508227e-6
1822: ----------------
1822: ##124     #:6   <== 4.99641
1822: ##124     #:6   ==> 4.99646
1822: @ Absolute error = 5.0000000000e-5, Relative error = 1.0007185159e-5

Reply:
I would think that you might run into problems with fewer digits as well (as the relative distance is larger). What happens if you increase the precision instead? The errors are tiny here as well. Maybe we need to either make the problem more difficult or use bigger time steps.